### DISCUSSION TITLE
Wiki unclear on custom CSS

### DISCUSSION THREAD

**Discussion by KyGost** [protz/thunderbird-conversations#1729] [Q&A]
As the altered message pane is in a shadowDOM, simply adding class names or the like to `userChrome.css` won't style it.

The wiki suggests that the pane *is* stylable: [Wiki - Tips and tricks: Disabling the "Quick Reply" feature](https://github.com/thunderbird-conversations/thunderbird-conversations/wiki/Tips-and-tricks:-Disabling-the-%22Quick-Reply%22-feature)
But, attempting simple changes as well as a variety of things I found in old closed issues, resulted in no changes.

Further, it does not seem as though this plugin uses any variables or parts which could be changed from a parent DOM style.

Clarification on whether this is possible or how would be much appreciated.

**Comment by Standard8** [comment]
I've just deleted the wiki page as it is out of date now. You can hide quick reply in the add-ons preferences.

Generally, I won't support userChrome.css - Conversations or Thunderbird may change things at any time which could break at any time.

I did fine this page about Firefox's new tab: https://windowspower.de/en/firefox-new-tab-page-more-entries-side-by-side/

So if you really want to you might be able to do something like that but for the stub page.
